infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/AWSProvider.java

```diff
@@ -91,10 +94,16 @@ public void init() throws TestGridInfrastructureException {
         String awsSecret = System.getenv(AWS_SECRET_ACCESS_KEY);
         if (StringUtil.isStringNullOrEmpty(awsIdentity) || StringUtil.isStringNullOrEmpty(awsSecret)) {
             throw new TestGridInfrastructureException(StringUtil
-                    .concatStrings("AWS Credentials must be set as environment variables: ", AWS_ACCESS_KEY_ID, ", ",
-                            AWS_SECRET_ACCESS_KEY));
+                    .concatStrings("AWS Credentials must be set as environment variables: ", AWS_ACCESS_KEY_ID,
+                            ", ", AWS_SECRET_ACCESS_KEY));
         }
         cfScriptPreprocessor = new CloudFormationScriptPreprocessor();
+        try {
+            awsAMIMapper = new AwsAMIMapper();
+        } catch (IOException e) {
```

Comment:
try-catch should wrap the wider code block, not to the single line where the exception is thrown. 

And, why would AwsAMIMapper constructor throw an IOException? That doesn't look right to me.. IMO, this shud not throw an exception